spring-integration-hazelcast/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Sat Feb 28 23:55:39 GMT 2015
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-bin.zip
```

Comment:
When you come up with the `build.gradle` based on that from DSL project, please re-run `wrapper` task to update this file as well for the `all` distribution.